fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/scalar/ScalarOperatorUtil.java

```diff
@@ -86,4 +88,20 @@ public static CallOperator buildMultiSumDistinct(CallOperator oldFunctionCall) {
                         FunctionSet.MULTI_DISTINCT_SUM, multiDistinctSum.getReturnType(),
                         oldFunctionCall.getChildren(), multiDistinctSum), DEFAULT_TYPE_CAST_RULE);
     }
+
+    public static CallOperator buildDateTrunc(List<ScalarOperator> arguments) {
```

Comment:
util method should be general.  if you put is in util class, this method should build many functions. otherwise, you should put this method in a single class.

Reply:
> util method should be general. if you put is in util class, this method should build many functions. otherwise, you should put this method in a single class.

removed from Util